WordPress/Classes/Extensions/Colors and Styles/WPStyleGuide+ApplicationStyles.swift

```diff
@@ -66,9 +66,6 @@ extension WPStyleGuide {
         UITabBar.appearance().unselectedItemTintColor = .tabUnselected
 
         let appearance = UITabBarAppearance()
-        appearance.configureWithOpaqueBackground()
```

Comment:
To achieve the default transparent scroll edge appearance, I think it needs to be either configured with a transparent background explicitly or this could that configured that tab bar could be removed all together.

Here's what happens if you remove this code and scroll to the bottom of a screen:


<img width="453" alt="Screenshot 2024-01-05 at 8 20 56 AM" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/1567433/33045497-599c-4280-a7af-10c964f6592e">